api/routes/api/events.go

```diff
@@ -60,3 +60,19 @@ func CreateEventHandler(c *gin.Context) {
 		"id":      id,
 	})
 }
+
+func ListEventsHandler(c *gin.Context) {
+	events, err := models.GetEvents()
```

Comment:
This takes too long as soon as the amount of events it too big. Perhaps add Pagination or alike to deal with it.